### PR TITLE
War with LanItems continues, another battle done

### PR DESCRIPTION
**Pull request by mchf**

### CODE REVIEW

**Review by imobachgs — APPROVED** [review]

File: src/lib/y2network/virtualization_config.rb [hunk 1]
```diff
@@ -72,14 +70,44 @@ def create
...
+        move_routes(builder.name, bridge_builder.name)
       end
 
       true
     end
 
   private
 
+    def yast_config
+      Y2Network::Config.find(:yast)
```
Comment (teclator) [inline src/lib/y2network/virtualization_config.rb:82]: Is this really needed?, as the config is an instance variable which is initialized by the constructor.

Comment (mchf) [reply src/lib/y2network/virtualization_config.rb]: right

File: src/lib/y2network/virtualization_config.rb [hunk 2]
```diff
@@ -72,14 +70,38 @@ def create
...
       end
 
       true
     end
 
   private
 
+    # Adds a new interface with the given name
+    def add_device_to_routing(name)
+      return if @config.nil?
```
Comment (teclator) [inline src/lib/y2network/virtualization_config.rb:83]: it is an attr_reader, you can use 

```suggestion
      return unless config
```
```suggestion
      return if @config.nil?
```

File: src/lib/y2network/virtualization_config.rb [hunk 3]
```diff
@@ -72,14 +70,38 @@ def create
...
+      return if @config.interfaces.any? { |i| i.name == name }
```
Comment (teclator) [inline src/lib/y2network/virtualization_config.rb:84]:
```suggestion
      return if config.interfaces.any? { |i| i.name == name }
```

Comment (teclator) [reply src/lib/y2network/virtualization_config.rb:84]: And the same with the rest

**Review by teclator — APPROVED** [review]
LGTM

**Pull request merged**